buildSrc/src/main/java/Dependencies.kt

```diff
@@ -85,6 +86,7 @@ object Dependencies {
     const val androidx_fragment = "androidx.fragment:fragment:${Versions.AndroidX.fragment}"
     const val androidx_lifecycle_extensions = "androidx.lifecycle:lifecycle-extensions:${Versions.AndroidX.lifecycle}"
     const val androidx_lifecycle_compiler = "androidx.lifecycle:lifecycle-compiler:${Versions.AndroidX.lifecycle}"
+    const val androidx_media = "androidx.media:media:${Versions.AndroidX.media}"
```

Comment:
I noticed  there is `androidx.media2`. Is 2x better? XD  
It looks like all the artifacts are on alpha stage.

Reply:
I looked at that too - but it's very undocumented and doesn't contain those the MediaSessionCompat classes. afaik